packages/gatsby-transformer-asciidoc/src/gatsby-node.js

```diff
@@ -116,6 +116,9 @@ const processPluginOptions = _.memoize((pluginOptions, pathPrefix) => {
     clonedPluginOptions.attributes = {}
   }
 
+  // Set the base_dir to the current node (i.e. document) path
+  clonedPluginOptions.base_dir = node.dir
```

Comment:
Usually, we don't use underscores. Change to `baseDir` for consistency?

Reply:
Sorry, I've been away from the Internet for a while camping.

This variable isn't a Gatsby variable, it's for Asciidoc. https://asciidoctor-docs.netlify.com/asciidoctor.js/processor/convert-options/

Reply:
Context is good. 

So this plugin is used to convert a file markedup with Asciidoc to HTML. One of the features of the language is the ability to include other snippets within a page. Asciidoc does this by converting `include::filename` to be `base_dir.include::filename` (that's the simplified version).

So that's why it is `base_dir` instead of `baseDir` which is my preference too. The `clonedPluginOptions` object is passed directly to the Asciidoc.js script, hence the spelling.